psalm-strict.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<psalm
+    totallyTyped="true"
```

Comment:
This setting is deprecated: https://psalm.dev/docs/running_psalm/configuration/#totallytyped